go.mod

```diff
@@ -5,11 +5,11 @@ go 1.16
 require (
 	cloud.google.com/go v0.47.0 // indirect
 	github.com/go-logr/logr v0.1.0
-	github.com/onsi/ginkgo v1.15.0
-	github.com/onsi/gomega v1.10.1
+	github.com/onsi/ginkgo v1.16.2
+	github.com/onsi/gomega v1.13.0
 	github.com/pkg/errors v0.9.1
-	github.com/talos-systems/cluster-api-bootstrap-provider-talos v0.2.0
-	github.com/talos-systems/talos/pkg/machinery v0.0.0-20210520203624-828772cec9a3
+	github.com/talos-systems/cluster-api-bootstrap-provider-talos v0.2.0-alpha.12.0.20210805142556-9fb0d07ca4d2
```

Comment:
this is not going backwards by the way, it's the `master` version now

(if that is going to confuse anyone)